e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -132,170 +133,174 @@ internal static class TestHelper {
         }
 
         /// <summary>
-        /// Save PublishedNodes json from OPC-PLC and provide the data to the tests
+        /// Update Device Twin tag
+        /// </summary>
+        /// <param name="patch">Name of deployed Industrial IoT</param>
+        /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
+        /// <param name="ct">Cancellation token</param>
+        public static async Task UpdateTagAsync(
+            string patch,
+            IIoTPlatformTestContext context,
+            CancellationToken ct = default
+        ) {
+            var registryManager = context.RegistryHelper.RegistryManager;
+            var twin = await registryManager.GetTwinAsync(context.DeviceConfig.DeviceId, ct);
+            await registryManager.UpdateTwinAsync(twin.DeviceId, patch, twin.ETag, ct);
+        }
+
+        /// <summary>
+        /// transfer the content of published_nodes.json file into the OPC Publisher edge module
         /// </summary>
-        /// <param name="simulatedOpcServer">Dictionary with URL of PLC-PLC as key and Content of Published Nodes files as value</param>
+        /// <param name="sourceFilePath">Source file path</param>
+        /// <param name="destinationFilePath">Destination file path</param>
         /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
-        public static void SavePublishedNodesFile(PublishedNodesEntryModel simulatedOpcServer, IIoTPlatformTestContext context) {
-            Assert.NotNull(simulatedOpcServer);
+        public static async Task PublishNodesInStandaloneMode(IEnumerable<PublishedNodesEntryModel> entries, IIoTPlatformTestContext context) {
```

Comment:
Please add `Async` in the end of method name and add `CancellationToken` parameter. The parameter should be passed to all async calls in the body.